Update sdk-in-browser.mdx (#1816)
SAP/cloud-sdk: docs-js/guides/sdk-in-browser.mdx

:::danger Warning

> Using SAP Cloud SDK for frontend use cases (in the browser) is not recommended in general, the tips shown in this doc might not work.

Author: ```suggestion
Using SAP Cloud SDK for frontend use cases (in the browser) is not recommended in general, the tips shown in this document are just work around and therefore might not work.
```

> Using SAP Cloud SDK for frontend use cases (in the browser) is not recommended in general, the tips shown in this document are just workaround and therefore might not work.

Reviewer: ```suggestion
Using the SAP Cloud SDK for frontend use cases (in the browser) is not recommended in general, the tips shown in this document are just workaround and therefore might not work.
```

> Using SAP Cloud SDK for frontend use cases (in the browser) is not recommended in general, the tips shown in this document are just workaround and therefore might not work.

Reviewer 2: ```suggestion
We recommend against using the SAP Cloud SDK for frontend use cases (in the browser). The following tips represent workarounds and are meant as informative examples.
```

> Using SAP Cloud SDK for frontend use cases (in the browser) is not recommended in general, the tips shown in this document are just work around and therefore might not work.

Author: ```suggestion
Using SAP Cloud SDK for frontend use cases (in the browser) is not recommended in general, the tips shown in this document are just workaround and therefore might not work.
```

:::

> Because of the specifics of a browser environment, some Cloud SDK features might be unavailable.

Author: ```suggestion
Because of the specifics of a browser environment, some SAP Cloud SDK features might be unavailable.
```
